### PR TITLE
Fix `AttributeError` in NWB export for `session_with_id`

### LINKED ISSUE #34: Exporting `session_with_id` to NWB results in an `AttributeError`
### Description
Calling `session_to_nwb` results in `AttributeError: 'NoneType' object has no attribute 'names'`. 

### Reproducibility
Include:
- OS (WIN | MACOS | Linux): Win
- DataJoint Element-Session Version: `0.1.5`
- MySQL Version: `8.0`
- MySQL Deployment Strategy (local-native | local-docker | remote): remote
- Minimum number of steps to reliably reproduce the issue: call `session_to_nwb( )` in a pipeline where `session_with_id` is activated.
- Complete error stack as a result of evaluating the above steps
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
[c:\Users\kusha\DJ-elements\vathes-team_devlab\notebooks\dj_to_nwb.ipynb](file:///C:/Users/kusha/DJ-elements/vathes-team_devlab/notebooks/dj_to_nwb.ipynb) Cell 18 line 3
      [1](vscode-notebook-cell:/c%3A/Users/kusha/DJ-elements/vathes-team_devlab/notebooks/dj_to_nwb.ipynb#X22sZmlsZQ%3D%3D?line=0) # element_lab_to_nwb_dict(lab_key=lab_key, protocol_key=protocol_key, project_key=project_key)
      [2](vscode-notebook-cell:/c%3A/Users/kusha/DJ-elements/vathes-team_devlab/notebooks/dj_to_nwb.ipynb#X22sZmlsZQ%3D%3D?line=1) subject_to_nwb(session_key=session_key)
----> [3](vscode-notebook-cell:/c%3A/Users/kusha/DJ-elements/vathes-team_devlab/notebooks/dj_to_nwb.ipynb#X22sZmlsZQ%3D%3D?line=2) session_to_nwb(session_key=session_key)
      [4](vscode-notebook-cell:/c%3A/Users/kusha/DJ-elements/vathes-team_devlab/notebooks/dj_to_nwb.ipynb#X22sZmlsZQ%3D%3D?line=3) imaging_session_to_nwb(session_key=session_key, processed_data_source="database", include_raw_data=False)

File [~\AppData\Roaming\Python\Python310\site-packages\element_session\export\nwb.py:58](https://file+.vscode-resource.vscode-cdn.net/c%3A/Users/kusha/DJ-elements/vathes-team_devlab/notebooks/~/AppData/Roaming/Python/Python310/site-packages/element_session/export/nwb.py:58), in session_to_nwb(session_key, lab_key, project_key, protocol_key, additional_nwbfile_kwargs)
     15 """Return subject and session metadata ass NWBFile object
     16 
     17 Gather session- and subject-level metadata and use it to create an NWBFile. If there
   (...)
     54     pynwb.NWBFile: NWB file object
     55 """
     57 # ensure only one session key is entered
---> 58 session_key = (session.Session & session_key).fetch1("KEY")
     60 session_identifier = {
     61     k: v.isoformat() if isinstance(v, datetime.datetime) else v
     62     for k, v in session_key.items()
     63 }
     65 nwbfile_kwargs = dict(
     66     session_id="_".join(session_identifier.values()),
     67     identifier=str(uuid4()),
     68 )
...
    198     )
    199     if not common_attributes:
    200         return not negate  # no matching attributes -> evaluates to True

AttributeError: 'NoneType' object has no attribute 'names'

```
### Expected Behavior
NWB file is created with session information. 

### Additional Research and Context
- [Original import](https://github.com/datajoint/element-session/blob/main/element_session/export/nwb.py#L5)
- [Alias](https://github.com/datajoint/element-session/blob/main/element_session/__init__.py) in the `__init__.py` causing the issue for `session_with_id`.

### PR DESCRIPTION
**Pull request by kushalbakshi**
This PR will close #34.

### CODE REVIEW

File: element_session/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-from . import session_with_datetime as session
+from . import session_with_id as session
```
Comment (kabilar) [inline element_session/__init__.py:1]: If I recall correctly, the original `session.Session` table had the `session_datetime` primary attribute.  So we added this line to maintain backwards compatibility.

File: element_session/export/nwb.py [hunk 2]
```diff
@@ -2,7 +2,13 @@
...
 import pynwb
 
-from .. import session
+from .. import session_with_id, session_with_datetime
+
+for session_module in [session_with_datetime, session_with_id]:
+    if session_module.schema.is_activated():
+        session = session_module
+    else:
+        continue
```
Comment (kabilar) [inline element_session/export/nwb.py:11]: Let's update this to handle the case where both modules are not activated.

File: element_session/__init__.py (hunk 1 above)
Comment (ttngu207) [reply element_session/__init__.py:1]: You're right, we need to keep that in place

Comment (kushalbakshi) [reply element_session/__init__.py:1]: Oh this makes sense. I'll revert the change in the next commit

File: element_session/export/nwb.py (hunk 2 above)
Comment (kushalbakshi) [reply element_session/export/nwb.py]: I've added a check before this step and it'll now raise an error if neither module is activated

File: element_session/export/nwb.py [hunk 3]
```diff
@@ -1,8 +1,21 @@
...
+    not session_with_id.schema.is_activated()
+    and not session_with_datetime.schema.is_activated()
+):
+    raise dj.DataJointError("Session schema has not been activated.")
+
+for session_module in [session_with_datetime, session_with_id]:
+    if session_module.schema.is_activated():
+        session = session_module
+    else:
+        continue
```
Comment (kabilar) [inline element_session/export/nwb.py:18]:
```suggestion
if session_with_datetime.schema.is_activated():
    session = session_with_datetime
elif session_with_id.schema.is_activated():
    session = session_with_id
else:
    raise dj.DataJointError("Session schema has not been activated.")
```

File: element_session/export/nwb.py [hunk 4]
```diff
@@ -5,18 +5,13 @@
...
-if (
-    not session_with_id.schema.is_activated()
-    and not session_with_datetime.schema.is_activated()
-):
+if session_with_datetime.schema.is_activated():
+    session = session_with_datetime
+elif session_with_id.schema.is_activated():
+    session = session_with_id
+else:
     raise dj.DataJointError("Session schema has not been activated.")
```
Comment (ttngu207) [inline element_session/export/nwb.py:14]: The for-loop approach for comparison - either way is fine

```
for session in (session_with_datetime, session_with_id):	
    if session.schema.is_activated():	
        break	
else:	
    raise dj.DataJointError("Session schema has not been activated.")
```

Comment (ttngu207) [reply element_session/export/nwb.py:14]: note, the `else` at the for-loop level

Comment (kushalbakshi) [reply element_session/export/nwb.py:14]: True. I forget that `for-else` is allowed in Python.

**Review by kabilar — APPROVED** [review]
Thank you, @kushalbakshi.

**Pull request merged**